packages/material-ui/src/LinearProgress/LinearProgress.js

```diff
@@ -177,7 +177,10 @@ function LinearProgress(props) {
     [classes.bar2Buffer]: variant === 'buffer',
   });
   const rootProps = {};
-  const inlineStyles = { bar1: {}, bar2: {} };
```

Comment:
since you are on it, would you mind making the same changes for circular progress as well?

Reply:
Sure. Should that be done in this or another branch?

Reply:
Just the same branch :-D

Reply:
Sorry. I commented on the wrong place. Now it's corrected. Take a look at my suggestions :-D